tests/installation/partitioning_raid.pm

```diff
@@ -202,12 +202,12 @@ sub modify_uefi_boot_partition {
     # fold the drive tree
     send_key 'left';
     send_key 'left' if is_storage_ng;    # With storage-ng first press folds vdd
-    assert_screen 'partitioning_raid-hard_disks-unfolded';
+
     # select first partition of the first disk (usually vda1), bit of a short-cut
     send_key 'right';
     # In storage ng other partition of the first disk can be selected, so select vda1 in the tree
     send_key 'right' if is_storage_ng;
-    assert_screen 'partitioning_raid-disk_vda_with_partitions-selected';
+    assert_screen 'partitioning_raid-disk_vda_with_partition1-selected';
```

Comment:
Would need renaming of all according needle tags, right?

Reply:
@okurz There was no such needle, so it should be fine.

Reply:
We have them for SLES at least, so we then need to change tag there. Not sure how it works with openSUSE without a single needle, as I would expect openQA to die saying that no candidate needle with tag found

Reply:
I have no access to SLES, so I cannot help here. 
When you have no needle, it does not match and die.

Reply:
I do not understand why you want to rename it. To be honest I don't see that as an improvement, otherwise I would do the job for SLE.

Reply:
@rwx788 the name is misleading. This is vda1 selected, not vda (or partitionS).

Reply:
We have this lvm+RAID1 on hyperv too, so sometimes it's sda not vda, but at least it's always first partition of the first disk. What would create less confusion IMHO would be something like "efi-partition-on-boot-disk-selected". WDYT?

Reply:
For the test, I also created ```partitioning_raid-disk_vda_with_partitions-selected``` so that openQA test go further (until the FAT part)

Reply:
@rwx788 ```efi-partition-on-boot-disk-selected``` sounds good. But let's rename it in another commit for coherency, no?

Reply:
@rwx788 I will create a separate PR once this one is accepted. But could you give me the name of additional needles which need to be renamed, please?